embassy-boot/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 edition = "2021"
 name = "embassy-boot"
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
Just heads up @TaylorCarpenter - On this project, David and I are adopting something we invented half a decade ago. That is, when we first make a change to forked upstream, we take the minor version, multiply it by 1000, then add one.

So, this would have been 0.2001.0.

After that, normal semver applies.

In this way:

1. We know the version from which we forked it.
2. It will never conflict with an upstream version in our lifetime.

Reply:
Oh cool. I didn't think of that.

Reply:
I had completely forgotten about it until David brought it up last week. But I recall it working well in the past.

Reply:
So how do we want to handle this? Should I push a new commit that rolls this version back?

Reply:
I've already handled it in my pulling of upstream's main. Just fix the last commit, then we merge this, then I have to redo my work slightly (there's a merge, so just rebasing doesn't work here) and my merge will fix it.

In the future, we *might* have some of these crates get clobbered by upstream's crates, but by then we shouldn't be using them anymore.